File: src/CSP-AF/CSPCore/CSPServices/CSPCloseBrowser.cs
using System;
using System.Collections.Generic;

using CloudSuitePortal;
using CloudSuitePortal.Services;

namespace actions.CSP
{
    class CSPCloseBrowser
    {
        public void run(Dictionary<string, object> Params)
        {
            try
            {
            	CSPBrowser.Application.WebDriver.Close();
            	CSPBrowser.Application.WebDriver.Quit();
            }
            catch {}
        }
    }
}

File: src/CSPAF/CSPCore/CSPServices/CSPCloseBrowser.java
package CSPAF.CSPCore.CSPServices;

import java.util.*;

import com.infor.cloudsuite.portal.core.services.*;

class CSPCloseBrowser
{
    public void run(HashMap<String, Object> params)
    {
        try
        {
            CSPBrowser.getApplication().getWebDriver().close();
           	CSPBrowser.getApplication().getWebDriver().quit();
        }
        catch (Exception e) {}
    }
}